src/main/java/codeu/controller/ProfileServerlet.java

```diff
@@ -0,0 +1,26 @@
+
+package codeu.controller;
+
+import java.io.IOException;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
```

Comment:
Typo in file name 
ProfileServerlet.java
should have been
ProfileServlet.java

Does this code compile? How did you test it?
Did you run
$ mvn test (https://sites.google.com/codeustudents.com/summer2018/reference-guides/testing)
Did all the test pass, meaning the code compiles successfully?
?

Reply:
I agree with Fang. This would likely cause an error, because in the web.xml file you are attempting to map the ProfileServlet.java file to the jsp file. It is a good idea to run 'mvn appengine:devserver' each time you make a change to make sure everything is running smoothly.